odoo/tools/_monkeypatches.py

```diff
@@ -1,14 +1,19 @@
 import ast
 import os
 import logging
-import num2words
 from odoo import MIN_PY_VERSION
-from .num2words_patch import Num2Word_AR_Fixed
 from shutil import copyfileobj
 from types import CodeType
 
 _logger = logging.getLogger(__name__)
 
+num2words = None
+try:
+    import num2words
+    from .num2words_patch import Num2Word_AR_Fixed
+except ImportError:
+    _logger.warning("num2words is not available, Arabic number to words conversion will not work")
```

Comment:
You can put the num2words = None here in the except, otherwise, it looks good to me

Reply:
I think it will trigger the CI/style, I modified to see if it's the case or not (I prefer your version)